packages/tx-construction/src/tx-builder/finalizeTx.ts

```diff
@@ -30,7 +30,7 @@ export const finalizeTx = async (
   { ownAddresses, witness, signingOptions, auxiliaryData, isValid, handles }: TxContext,
   { inputResolver, keyAgent }: FinalizeTxDependencies,
   stubSign = false
-): Promise<SignedTx> => {
+): Promise<SubmitTxArgs> => {
```

Comment:
We're changing this type to `SubmitTxArgs` but the type of `SubmitTxArgs['signedTransaction']` is meant to be an `OpaqueString`, here we're returning an object of type `Cardano.Tx`. Are we meant to be serializing the `signedTransaction` here, or is the type incorrect? @mkazlauskas

Reply:
b15261eddb78a02ef40bd9688f6f5792babe75e9 should fix this.